README.md

```diff
@@ -7,9 +7,10 @@ Please submit a pull request to make additions or changes.
 | Company       | Description   |
 | ------------- | ------------- |
 | [Advanced Microgrid Solutions](http://www.advmicrogrid.com/careers.html) | Advanced Intelligence for Energy Markets |
+| [Amperon](https://amperon.co/)| Amperon provides real time load forecasts and analytics based on high-resolution smart meter data for energy suppliers and utilities. |
```

Comment:
Hesitant to add this one since it currently goes to a landing page devoid of information or contact means. Is there another place they currently have jobs listed or something?

Reply:
Hmm, not that I could find.  I really only know about them because a friend is directly involved.  I'll remove for now.

Thanks for keeping a high standard here!

Reply:
Yup, just trying to keep the grunt work and duds to a minimum for interested folks browsing. Happy to add it back as soon as they've got something of course.